app/src/main/res/layout-land/coming_soon_topic_list.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.home.promotedlist.ComingSoonTopicListViewModel" />
+  </data>
+
+  <LinearLayout
```

Comment:
Just for consistency, we should use constraint layout @rt4914 @BenHenning

Reply:
Deferring it to @rt4914 @BenHenning

Reply:
Done.